src/Azure.Functions.Cli/Common/Utilities.cs

```diff
@@ -17,6 +17,7 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Abstractions;
+using static Azure.Functions.Cli.Common.OutputTheme;
```

Comment:
nit: in this case, for a single use, I'm not sure this is providing much benefit.